web/elm/src/Dashboard/Group.elm

```diff
@@ -178,56 +212,71 @@ viewFavoritePipelines session params =
         )
 
 
-tag : { a | userState : UserState } -> Group -> Maybe Tag.Tag
-tag { userState } g =
+tag : { a | userState : UserState } -> Concourse.TeamName -> Maybe Tag.Tag
+tag { userState } teamName =
     case userState of
         UserStateLoggedIn user ->
-            Tag.tag user g.teamName
+            Tag.tag user teamName
 
         _ ->
             Nothing
 
 
 hdView :
     { pipelineRunningKeyframes : String
-    , pipelinesWithResourceErrors : Set ( String, String )
-    , pipelineJobs : Dict ( String, String ) (List Concourse.JobIdentifier)
-    , jobs : Dict ( String, String, String ) Concourse.Job
+    , pipelinesWithResourceErrors : Set Concourse.DatabaseID
+    , pipelineJobs : Dict Concourse.DatabaseID (List Concourse.JobName)
+    , jobs : Dict ( Concourse.DatabaseID, Concourse.JobName ) Concourse.Job
+    , dashboardView : Routes.DashboardView
+    , query : String
     }
     -> { a | userState : UserState }
-    -> Group
+    -> ( Concourse.TeamName, List Card )
     -> List (Html Message)
-hdView { pipelineRunningKeyframes, pipelinesWithResourceErrors, pipelineJobs, jobs } session g =
+hdView { pipelineRunningKeyframes, pipelinesWithResourceErrors, pipelineJobs, jobs, dashboardView, query } session ( teamName, cards ) =
```

Comment:
Now that we pass in the `session`, we can probably grab the `pipelineRunningKeyframes` from there, let's move it to the second argument it'll make it slightly cleaner https://github.com/concourse/concourse/blob/94af3c750b0ccc28931054605a81fc0972e6c2e3/web/elm/src/Dashboard/Dashboard.elm#L1423-L1432